### DISCUSSION TITLE
How to implement multiple env file in prisma graphql

### DISCUSSION THREAD

**Discussion by kavibartwal** [prisma/prisma#14363] [Q&A]
hi there,

I am using prisma in graphql there i want to use multiple env file for prisma, .env.test for QA database and .env.development for prod database.
This is my folder structure:
![image](https://user-images.githubusercontent.com/107266439/179549928-627f1464-1f15-4238-8bfd-ec57b5a7104c.png)
Right now by default prisma is using .env and taking environment variable from there and when i am changing .env file to .env.test and .env.development , graphql give me below attached error.
![image](https://user-images.githubusercontent.com/107266439/179550519-875c4aa8-8cf6-4245-88c8-6b8254079788.png)

This is my package.json:
scripts:
{
 "**build**": "dotenv -e .env.test -- yarn run clean && dotenv -e .env.test -- babel src -d dist &&  dotenv -e .env.test -- prisma generate",
 "**start:local**": "dotenv -e .env.test -- yarn run build && dotenv -e .env.test -- node --max-old-space-size=4096 ./dist/server/localServer.js"
}
i have already tried the multiple env files method mentioned is prisma doc https://www.prisma.io/docs/guides/development-environment/environment-variables/using-multiple-env-files but did not get any success.

**Comment by well-balanced** [comment]
could you share your full folder structure? (including `package.json`)

I guess that the wrong path to the `.env.test` is the cause.
how about change your path from `dotenv -e .env.test` to `dotenv -e src/prisma/.env.test`?

**Reply by kavibartwal** [reply]
Thanks it worked for me.

**Reply by well-balanced** [reply]
happy hack :)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!

**Comment by gaojie0910** [comment]
use fs copy .env.dev .env.test etc to .env when node mode change
